_posts/2021-03-23-connector-build.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: post
+title:  "Building your own Kafka Connect image with a custom resource"
+date: 2021-03-23
+author: jakub_stejskal
+---
+In a [previous blog post](https://strimzi.io/blog/2020/05/07/camel-kafka-connectors/) we showed how easy it is to integrate [Camel Kafka Connectors](https://camel.apache.org/camel-kafka-connector/latest/) with Strimzi by configuring a `KafkaConnect` custom resource.
+That approach had one limitation - you had to build your own Kafka Connect image and use it in the custom resource.
+This step is no longer needed thanks to a feature introduced in Strimzi 0.22 that allows custom Kafka Connect images to be built directly by Strimzi.
+
```

Comment:
We need a `<!-- more -->` somewhere to Jekyll knows where to terminate the summary.